src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.h

```diff
@@ -123,20 +124,22 @@ class Delegate
      *   method returns `CHIP_NO_ERROR`. If an error is returned, `outSession` is left unmodified or
      *   set to an invalid state.
      *
-     * @param[in] sessionHandle
-     *   The secure session handle representing the connection with the requestor.
+     * @param[in]  peerId
+     *   Identifies the peer (requestor) connection over which this command was received. The delegate can
+     *   use this `PeerId` to retrieve or track any existing secure session state as needed.
      *
      * @param[in] originatingEndpointId
-     *   The endpoint ID from which the ProvideOffer command was received. This can be used to to invoke
-     *   the commands on the Requestor cluster.
+     *   This field SHALL indicate the endpoint that originates the ProvideOffer command.
```

Comment:
Again: it's not clear that it has to do anything with where the command originated.  It just says where the commands we send should go.

Nothing in the cluster spec requires the server for the Requestor to live on the same endpoint as the client for the Provider.